core/src/main/java/com/linecorp/armeria/server/ServerBuilder.java

```diff
@@ -177,6 +177,7 @@
     private Duration gracefulShutdownQuietPeriod = DEFAULT_GRACEFUL_SHUTDOWN_QUIET_PERIOD;
     private Duration gracefulShutdownTimeout = DEFAULT_GRACEFUL_SHUTDOWN_TIMEOUT;
     private Executor blockingTaskExecutor = CommonPools.blockingTaskExecutor();
+    private boolean shutdownBlockingTaskExecutorOnStop;
```

Comment:
I wonder if we need to add this to `Flags`, WDYT? /cc @trustin

Reply:
I'm not sure I like the idea. See my comment at: https://github.com/line/armeria/pull/1707#discussion_r277188573